extensions/parcel/src/my-deliveries.tsx

```diff
@@ -46,57 +46,75 @@ export default function Command() {
     return `(in ${daysUntil} day${daysUntil !== 1 ? "s" : ""})`;
   };
 
-  // Format date in a more readable way: "Feb 26, 2025"
-  const formatFriendlyDate = (dateString: string): string => {
-    if (!dateString) return "Unknown date";
-
+  // Utility function to parse dates in various formats
```

Comment:
```suggestion
```

The original comment gave a hint about the return string's format. Here, the type signature tells you everything you need, the comment is duplicative.